vdslib/src/vespa/vdslib/distribution/distribution_config_bundle.h

```diff
@@ -0,0 +1,48 @@
+// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+#pragma once
+
+#include "bucket_space_distribution_configs.h"
+#include "distribution.h"
+#include <iosfwd>
+
+namespace storage::lib {
+
+class DistributionConfigBundle {
```

Comment:
Consider adding class comment.

Reply:
Will be done as part of a follow-up; this code is currently in motion 🏃‍➡️